src/main/kotlin/com/bugsnag/android/gradle/internal/GitVersionValueSource.kt

```diff
@@ -0,0 +1,22 @@
+package com.bugsnag.android.gradle.internal
+
+import org.gradle.api.provider.ValueSource
+import org.gradle.api.provider.ValueSourceParameters
+import org.gradle.process.ExecOperations
+import java.io.ByteArrayOutputStream
+import java.nio.charset.Charset
+import javax.inject.Inject
+
+abstract class GitVersionValueSource : ValueSource<String, ValueSourceParameters.None> {
+    @get:Inject
+    abstract val execOperations: ExecOperations
+
+    override fun obtain(): String {
+        val output = ByteArrayOutputStream()
+        execOperations.exec {
+            it.commandLine("git", "--version")
+            it.standardOutput = output
```

Comment:
Does this handle the case when the user doesn't have `git` installed? Although I'm not entirely sure if the old way of calling it did either.